src/config/nexarear_squeezeDetPlus_config.py

```diff
@@ -0,0 +1,87 @@
+# --------------------------------------------------------------------------------------------------
+#
+#   Copyright (c) 2016, 2017. Nexar Inc. - All Rights Reserved. Proprietary and confidential.
+#
+#   Unauthorized copying of this file, via any medium is strictly prohibited.
+# --------------------------------------------------------------------------------------------------
+
+"""Model configuration for nexarear dataset"""
+
+import numpy as np
+
+from config import base_model_config
+
+def nexarear_squeezeDetPlus_config():
+  """Specify the parameters to tune below."""
+  mc                       = base_model_config('NEXAREAR')
+
+  mc.IMAGE_WIDTH           = 1280
+  mc.IMAGE_HEIGHT          = 720
+  mc.BATCH_SIZE            = 20
+
+  mc.WEIGHT_DECAY          = 0.0001
+  mc.LEARNING_RATE         = 0.01
+  mc.DECAY_STEPS           = 10000
+  mc.MAX_GRAD_NORM         = 1.0
+  mc.MOMENTUM              = 0.9
+  mc.LR_DECAY_FACTOR       = 0.5
+
+  mc.LOSS_COEF_BBOX        = 5.0
+  mc.LOSS_COEF_CONF_POS    = 75.0
+  mc.LOSS_COEF_CONF_NEG    = 100.0
+  mc.LOSS_COEF_CLASS       = 1.0
+
+  mc.PLOT_PROB_THRESH      = 0.4
+  mc.NMS_THRESH            = 0.4
+  mc.PROB_THRESH           = 0.005
+  mc.TOP_N_DETECTION       = 64
+
+  mc.DATA_AUGMENTATION     = True
+  mc.DRIFT_X               = 150
+  mc.DRIFT_Y               = 100
+  mc.EXCLUDE_HARD_EXAMPLES = False
+
+  mc.ANCHOR_BOX            = set_anchors(mc)
+  mc.ANCHORS               = len(mc.ANCHOR_BOX)
+  mc.ANCHOR_PER_GRID       = 9
+
+  mc.PERCENTAGE_OF_TEST_SET = 0.3
+  mc.RANDOM_SEED_TEST_TRAIN_SPLIT = 314
+
+  return mc
+
+def set_anchors(mc):
+  H, W, B = 43, 78, 9
+  anchor_shapes = np.reshape(
+      [np.array(
```

Comment:
Not really. The anchor boxes are the same for squeezeDet and squeezeDet+ but different for resnet